pages/delegate-detail.vue

```diff
@@ -62,11 +62,78 @@
       </el-row>
     </b-card>
 
+    <b-card :title="blockTitle" class="shadow mt-4">
+      <el-table :data="currentBlocks" stripe style="width: 100%; margin: auto;" v-loading="loading">
+        <el-table-column prop="height" align="center" label="Height" width="80">
+          <template v-slot:default="table">
+            <nuxt-link class="nuxt-link" :to="{name: 'block-detail', query: { height: table.row.height }}">
+              {{table.row.height}}
+            </nuxt-link>
+          </template>
+        </el-table-column>
+        <el-table-column prop="id" align="center" label="Block ID" width="100">
+          <template v-slot:default="table">
+            <nuxt-link class="nuxt-link" :to="{name: 'block-detail', query: { height: table.row.height }}">
+              {{subID(table.row.id)}}
+            </nuxt-link>
+          </template>
+        </el-table-column>
+        <el-table-column prop="timestamp" align="center" label="Forged Time" width="170" :formatter="timestamp2date"></el-table-column>
+        <el-table-column prop="count" align="center" label="Transactions" width="110"></el-table-column>
+        <el-table-column prop="fees" align="center" label="Fees" width="130" :formatter="formatFees"></el-table-column>
+        <el-table-column prop="reward" align="center" label="Reward" width="90" :formatter="formatReward"> </el-table-column>
+        <el-table-column prop="delegate" align="center" label="Delegate" width="150">
+          <template v-slot:default="table">
+            <nuxt-link class="nuxt-link" :to="{name: 'delegate-detail', query: { publicKey: table.row.delegate }}">
+              {{subDelegate(table.row.delegate)}}
+            </nuxt-link>
+          </template>
+        </el-table-column>
+      </el-table>
+      <el-pagination
+        @current-change="handleCurrentChange"
+        :current-page="currentPage"
+        :page-size="10"
+        layout="prev, pager, next"
+        :total="blocksCount"
+        align="center"
+      ></el-pagination>
+    </b-card>
+
+    <b-card title="Voters" class="shadow mt-4">
+      <el-table :data="voters" stripe style="width: 100%; margin: auto;" v-loading="voteLoading">
+        <el-table-column prop="senderId" align="center" label="Address">
+          <template v-slot:default="table">
+            <nuxt-link class="nuxt-link" :to="{name: 'account-detail', query: { address: table.row.address }}">
```

Comment:
This doesn't get showed

Example: https://deploy-preview-97--testnet-explorer.netlify.app/delegate-detail?username=tonyt_gny

![image](https://user-images.githubusercontent.com/11912239/112300932-b20ccb80-8c99-11eb-9165-f8e226166a23.png)